src/StreetNameRegistry.Api.BackOffice/Infrastructure/Modules/ApiModule.cs

```diff
@@ -76,7 +77,10 @@ protected override void Load(ContainerBuilder builder)
 
             builder.RegisterSnapshotModule(_configuration);
 
-            _services.AddAcmIdmAuthorizationHandlers();
+            // Authorization
+            _services
+                .AddAcmIdmAuthorizationHandlers()
+                .AddNisCodeAuthorizationWithDynamoDb();
```

Comment:
Don't forget to use hardcoded registration when environment is development.

Reply:
See last commit.